src/api/xcm.ts

```diff
@@ -0,0 +1,128 @@
+import { assetsMap, chainsConfigMap, chainsMap } from "@galacticcouncil/xcm-cfg"
+import { ConfigService, SubstrateApis } from "@galacticcouncil/xcm-core"
+import { Wallet } from "@galacticcouncil/xcm-sdk"
+import { useMutation, useQuery } from "@tanstack/react-query"
+import { Transaction, useStore } from "state/store"
+import { isAnyParachain } from "utils/helpers"
+import { QUERY_KEYS } from "utils/queryKeys"
+import { external } from "@galacticcouncil/apps"
+import { ASSETHUB_XCM_ASSET_SUFFIX } from "./external/assethub"
+import { TRegisteredAsset } from "sections/wallet/addToken/AddToken.utils"
+import { useTranslation } from "react-i18next"
+import { createToastMessages } from "state/toasts"
+
+type TransferProps = {
+  asset: string
+  srcAddr: string
+  srcChain: string
+  dstAddr: string
+  dstChain: string
+}
+
+export const xcmConfigService = new ConfigService({
+  assets: assetsMap,
+  chains: chainsMap,
+  chainsConfig: chainsConfigMap,
+})
+
+export const wallet = new Wallet({
+  config: xcmConfigService,
+})
+
+export const createXcmAssetKey = (id: string, symbol: string) => {
+  if (id && symbol) {
```

Comment:
why is it here since id an symbol are required arguments?

Reply:
fixed